src/org/usfirst/frc4904/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,66 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.Util;
+import org.usfirst.frc4904.standard.commands.Idle;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.standard.custom.sensors.CustomEncoder;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.PositionSensorMotor;
+import edu.wpi.first.wpilibj.SpeedController;
+
+public class Arm extends PositionSensorMotor {
+	public static final double ELBOW_MULTIPLIER = 1.0;
+	public final Motor elbowMotorA;
```

Comment:
Remove the instance fields/members/variables/whatevers `elbowMotorA`, `elbowMotorB`, `elbowMotionController`.